### ISSUE TITLE
Wide Review Comment 2017: hard to understand because of terminology

### ISSUE THREAD

**Issue by nigelmegitt** [w3c/webvtt#381]
Copy/paste from https://lists.w3.org/Archives/Public/public-tt/2017Sep/0080.html - raising as an issue for tracking/disposition purposes.

It is extremely difficult to read and understand the WebVTT specification because of the terminology used, especially in relation to positioning. Many concepts have overlapping use of words when they relate to different things. For example section 4.4 "WebVTT cue settings" includes the "WebVTT line cue setting" which actually configures the "cue box" position.

**Comment by silviapfeiffer** [comment]
@nigelmegitt we had this discussion about words used before, see https://www.w3.org/Bugs/Public/show_bug.cgi?id=28073  - unless you have a better proposal for what to use, I don't see how I can address this.

About your specific example: all the cue settings configure the position and alignment of the cue box and the text within, as stated in https://www.w3.org/TR/webvtt1/#cue-settings .

If your problem is the repetition of the word "WebVTT", then that may indeed be something to remove since we're looking at the WebVTT spec. Is that your problem?

**Comment by nigelmegitt** [comment]
Thanks for the pointer @silviapfeiffer . I think yes, removing WebVTT would be okay, though it was not the problem I was reporting. 

The problem I'm highlighting is that some of the labels don't seem to relate to the thing they are labels for. Take "line cue setting" - it seems like it should be the setting for a cue line, but it isn't, it's the position for a cue box. So it should be called "cue box position" or something like that. Now "position", which as a word in general use does not have any connotation of axis, is being (mis)used to be a position specifically in the dimension orthogonal to the line, so that isn't actually available. Even "cue box line" would be slightly, though not much, better.

**Comment by silviapfeiffer** [comment]
Did you look back over that issue and the discussion there?

"Line" referees to the offset from the top and "position" to the offset from the left (for simplicity I'm referring to horizontally rendered cues). It has been this way from the start of WebVTT.

A cue line setting would not be called " line cue setting", but " cue line setting". It's the "line" cue setting that is being referred there. Maybe putting quotes around the name of the cue setting helps. It seems you're confusing reading English with reading the algorithmically written specification.

**Comment by nigelmegitt** [comment]
@silviapfeiffer Yes, I did. The bugzilla issue and discussion seems to mirror my reading as well - with the same problems identified but not really resolved to anyone's satisfaction.

> It seems you're confusing reading English with reading the algorithmically written specification.

No, I'm not. You could call the labels anything ("frog", "robert", "toaster" etc) and it would still work as a formal spec, but we don't do that because it makes it harder to read. I'm suggesting that we need to use labels that make it easier to read because psychology is a thing and humans are human and specs are better if they are easier to read, other things being equal.

The idea of a drawing was a good one by the way, for exactly this reason.

Putting some formatting around the key point as you suggest, like quotes, might also help. For example:
* "line" cue setting
* _line_ cue setting

I still think that the concepts of the cue box settings and the cue settings as applicable within that cue box should be separated and clarified.

**Comment by dwsinger** [comment]
remove superfluous words (e.g. webvtt) and try to put the words in the right order.  we agree to clarify as we can.

**Comment by silviapfeiffer** [comment]
Resolution at TPAC: remove "WebVTT" everywhere, add formatting to improve readability

**Comment by nigelmegitt** [comment]
also please check back to make sure those settings tie back with the descriptive text explaining the box model agreed in #383.
